packages/components/src/styles/footnotes/footnotes.stories.ts

```diff
@@ -55,39 +52,47 @@ export const Default = {
 };
 
 /**
- * Use the class with a `ul` element to display the footnotes as an unordered list.
+ * Use the standard html list elements for alternative appearances:
+ *
+ * - numbered: use an `ol` element to create a numbered list.
+ * The html `start` attribute to set the starting number of the list, the default value is '1'.
+ * - unnumbered: use an `ul` element to create an unnumbered list.
  */
 
-export const UnorderedLists = {
-  parameters: { controls: { include: [] } },
-
-  render: (args: any) => {
-    return generateTemplate({
-      options: {
-        templateContent: '<div class="sd-container"><ul class="%CLASSES%">%SLOT%</ul></div?'
-      },
-      args
-    });
-  }
+export const Variants = {
+  render: () => html`
+    <div class="flex flex-col gap-12">
+      <ol class="sd-footnotes">
+        <li>Lorem ipsum dolor sit amet, consetetur sadipscing elitr.</li>
+        <li>Sed diam nonumy eirmod tempor invidunt ut labore.</li>
+        <li>Dolore magna aliq erat, sed diam voluptua.</li>
+      </ol>
+      <ol start="150" class="sd-footnotes">
+        <li>Lorem ipsum dolor sit amet, consetetur sadipscing elitr.</li>
+        <li>Sed diam nonumy eirmod tempor invidunt ut labore.</li>
+        <li>Dolore magna aliq erat, sed diam voluptua.</li>
+      </ol>
+      <ul class="sd-footnotes">
+        <li>Lorem ipsum dolor sit amet, consetetur sadipscing elitr.</li>
+        <li>Sed diam nonumy eirmod tempor invidunt ut labore.</li>
+        <li>Dolore magna aliq erat, sed diam voluptua.</li>
+      </ul>
+    </div>
+  `
 };
 
 /**
- * This sample shows how to use the `start` attribute to set the starting number of the list. This is a html standard attribute.
+ * Use the `sd-footnotes--inverted` class to invert the color of the footnotes.
  */
 
-export const Sample = {
-  parameters: { controls: { include: [] } },
-  name: 'Sample: Set start',
-  render: (args: any) => {
-    return generateTemplate({
-      options: {
-        templateContent: `<div class="sd-container">
-        <ol start="50" class="%CLASSES%">%SLOT%</ol>
-        <ol start="150" class="%CLASSES%">%SLOT%</ol>
-        <ol class="%CLASSES%">%SLOT%</ol>
-      </div>`
-      },
-      args
-    });
-  }
+export const Inverted = {
+  render: () => html`
+    <div class="sd-container sd-container--variant-primary">
+      <ol class="sd-footnotes sd-footnotes--inverted">
```

Comment:
In figma our inverted variant is primary-400.
in code you have white?
if yes we need to create a fix ticket to make it primary.400